containers/ddev-webserver/files/etc/skel/.bashrc

```diff
@@ -111,3 +111,11 @@ if ! shopt -oq posix; then
     . /etc/bash_completion
   fi
 fi
+
+# set COMPOSER_BIN_DIR to $WEBSERVER_DOCROOT/vendor/bin if unset
+if [[ ! -v $COMPOSER_BIN_DIR ]]; then
+    export COMPOSER_BIN_DIR="$WEBSERVER_DOCROOT/vendor/bin"
+fi
+
+# add composer bin dir to PATH
+export PATH="$COMPOSER_BIN_DIR:$PATH"
```

Comment:
This should not be at the beginning of the $PATH, it should be at the end IMO.

Reply:
IMHO it should be at the start, because if there is an executable in both directories with the same name you explicitly installed it to be used instead of whatever is in the rest of the path.

Reply:
The problem is that you're changing behavior for people that way. For example, on a Drupal 8 site you'll get vendor/bin/drush instead of /usr/local/bin/drush. I'm more or less OK with this PR if we don't create any surprises for people, but not OK with it if it changes behavior.